docker/app/tezos/dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM tezos/tezos:alphanet
+FROM tezos/tezos:protocol
```

Comment:
Placeholders usually have `$` as a prefix, e.g. `$protocol`.

Reply:
$ would make it a variable, I'm using sed to replace the string in place.